src/context-parser.js

```diff
@@ -968,7 +971,7 @@ function DisableIEConditionalComments(state, i){
         // for lazy conversion
         this._convertString2Array();
 
-        this.input.splice(i, 0, ' ');
+        this.input.splice(i + 1, 0, ' ');
```

Comment:
Was a bug. The initial commit is to add a space before ]>. When i is incremented, ]> is encountered again, and that caused an endless loop.